apps/meteor/app/slashcommands-join/server/server.ts

```diff
@@ -1,53 +1,54 @@
 import { Meteor } from 'meteor/meteor';
 import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
-import { IMessage } from '@rocket.chat/core-typings';
 
 import { Rooms, Subscriptions } from '../../models/server';
 import { settings } from '../../settings/server';
 import { slashCommands } from '../../utils/lib/slashCommand';
 import { api } from '../../../server/sdk/api';
 
-function Join(_command: 'join', params: string, item: IMessage): void {
-	let channel = params.trim();
-	if (channel === '') {
-		return;
-	}
-
-	channel = channel.replace('#', '');
-
-	const userId = Meteor.userId() as string;
-	const user = Meteor.users.findOne(userId);
-	const room = Rooms.findOneByNameAndType(channel, 'c');
-
-	if (!user) {
-		return;
-	}
-
-	if (!room) {
-		api.broadcast('notify.ephemeralMessage', userId, item.rid, {
-			msg: TAPi18n.__('Channel_doesnt_exist', {
-				postProcess: 'sprintf',
-				sprintf: [channel],
-				lng: settings.get('Language') || 'en',
-			}),
+slashCommands.add(
+	'join',
+	(_command: 'join', params, item): void => {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.